### PR TITLE
Thank you contributors

### PR DESCRIPTION
**Pull request by version0chiro**
Added a material UI based thank you card at the bottom of the page. Also extracted the current count of contributors using GitHub's API and displayed top 6 avatars who have contributed to the project. This is how the card looks.
 
image

I have added a separate component for the card for easy editing and reviewing. Kindly check the  Contributors card in the front end directory. I am unsure how to proceed forward with the building of front end to the server, would be great if someone can help me with that, thank you.

### CODE REVIEW

**Comment by shravan20** [comment]
Hi @version0chiro 

Thank you for your time to contribute towards our feature request. We will review your code, and accept the PR.

Also, will you please add the screenshot of the whole screen with this component?

**Comment by version0chiro** [comment]
Hey @shravan20 , this is how the screen looks with the component, I was a bit unsure about the placement but can easily change if you have any specific place you wanna keep it at.
image

**Comment by shravan20** [comment]
Hi @version0chiro 
Good job on the design!! 😄 
Thank you for taking up time for contributing towards the project! We value your contribution! 👏🏻 
Cheers!

File: frontend/src/components/ContributorsCard/ContributorCard.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  },
+  pos: {
+    marginBottom: 12,
+  },
+});
+
+const ContributorsCard = () => {
+  const [listOfContributors,setListOfContributors] = useState([]);
+  useEffect(()=>{
+      fetch("https://api.github.com/repos/shravan20/github-readme-quotes/contributors?")
```
Comment (shravan20) [inline frontend/src/components/ContributorsCard/ContributorCard.js:34]: Can you move the URL to a urlConfig.js file and export it from there under a new folder called /constants?? This is to ensure that we maintain code quality. Please make the changes by evening, we will have it accepted by evening.

**Review by shravan20 — CHANGES_REQUESTED** [review]
Minor changes requsted.

File: frontend/src/components/ContributorsCard/ContributorCard.js (hunk 1 above)
Comment (version0chiro) [reply frontend/src/components/ContributorsCard/ContributorCard.js:34]: okay no problem in that, can i make the folder inside the Contributors Card or do you want it to be in the components?

Comment (shravan20) [reply frontend/src/components/ContributorsCard/ContributorCard.js:34]: Create a folder /frontend/src/CONSTANTS. 
Create a file urlConfig.js. Just move URL to that!
 
Rest of the things looks neat!

Comment (version0chiro) [reply frontend/src/components/ContributorsCard/ContributorCard.js]: I have exported the URL from a new file, kindly check the changes in the latest commit

**Review by shravan20 — APPROVED** [review]
Changed Accepted : PR

**Pull request merged**